### ISSUE TITLE
Ask for some details

### ISSUE THREAD

**Issue by cvv-student** [VITA-Group/GAN-Slimming#19]
What an exciting work! I noticed two details when running the code. 
(1) Whether it is GS or fine-tune, G and D use a smaller learning rate, while the norm factor gamma uses a larger learning rate. Why? 
(2) When running gs.py, only dataset horse2zebra ( in 104 lines) needs to load the pre training model and retrain. Why?

**Comment by cvv-student** [comment]
Besides, I'm curious about how to control the pruning ratio.
